app/views/layouts/posts/index.html.erb

```diff
@@ -0,0 +1 @@
+<section>Hello from user's all posts page!</section>
```

Comment:
- [ ] This file is in the `app/views/layouts/posts`, whereas it should be `app/views/posts`. Kindly move this file to the latter directory.